docker-compose-down.sh

```diff
@@ -1 +1 @@
-docker compose down
\ No newline at end of file
+docker-compose down
```

Comment:
Corporate Mac is Podman so doesn't have a `docker compose` and uses the legacy `docker-compose`